ironfish/src/workerPool/serializableMessage.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can we rename this file to `workerMessage` now the class is renamed?

Reply:
Done. 💯